### PR TITLE
[DOCS] Spell check and minor tidying up of Infrastructure Monitoring Guide

### PR DESCRIPTION
**Pull request by Titch990**
Minor changes and reorg, for consistency and logic.
There are more changes to follow, and some outstanding questions to resolve, for which I'll raise separate issues as necessary.

Part of https://github.com/elastic/observability-dev/issues/104.

Hi @elastic/infrastructure. Could I get a review on this, please? Docs build can be seen here http://stack-docs_476.docs-preview.app.elstc.co/guide/en/infrastructure/guide/master/index.html

### CODE REVIEW

**Comment by Titch990** [comment]
Pinging @bmorelli25 and @elastic/infrastructure. Could I have a review, please?

File: docs/en/infraops/installation.asciidoc [hunk 1]
```diff
@@ -1,165 +1,120 @@
...
-configuration changes in response to changes in your containers. To learn how,
-see:
-
-* {filebeat-ref}/configuration-autodiscover.html[{filebeat} autodiscover
-configuration]
-* {metricbeat-ref}/configuration-autodiscover.html[{metricbeat} autodiscover
-configuration]
+=== Running Metricbeat 6.5 or earlier with Kibana 6.6 or later
 
+// ++ Is this needed here since we've just installed the latest version of both Kibana and Metricbeat?
```
Comment (jsoriano) [inline docs/en/infraops/installation.asciidoc]: We may remove this section if it is intended for latest versions, Beats < 6.8 don't work with ES/Kibana 7.X.

**Comment by Titch990** [comment]
@jsoriano. Thanks for taking the time to give such detailed answers. Perhaps I should have been a bit clearer -  the comments were really placeholders for me to ensure I picked up the more detailed issues at some point. For now, I simply wanted to get some updated content out there. But I've got some answers too now! Thank you :-)

File: docs/en/infraops/infrastructure-metrics.asciidoc [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+
+*CPU Usage*:: Average of `system.cpu.user.pct` added to the average of `system.cpu.system.pct` divided by `system.cpu.cores`
+
+*Memory Usage*:: Average of `system.memory.actual.used.pct`
+
+*Load*:: Average of `system.load.5`
+
+*Inbound Traffic*:: Derivative of the maximum of `system.network.in.bytes` scaled to a 1 second rate
+
+*Outbound Traffic*:: Derivative of the maxoimum of `system.network.out.bytes` scaled to a 1 second rate
```
Comment (bmorelli25) [inline docs/en/infraops/infrastructure-metrics.asciidoc]:
```suggestion
*Outbound Traffic*:: Derivative of the maximum of `system.network.out.bytes` scaled to a 1 second rate
```

File: docs/en/infraops/infrastructure-metrics.asciidoc [hunk 3]
```diff
@@ -0,0 +1,54 @@
...
+This metric relies on the same indices as the logs.
+
+[float]
+=== Docker Container Metrics
+
+*CPU Usage*:: Average of `docker.cpu.total.pct`
+
+*Memory Usage*:: Average of `docker.memory.usage.pct`
+
+*Inbound Traffic*:: Derivative of the max of `docker.network.in.bytes` scaled to a 1 second rate
```
Comment (bmorelli25) [inline docs/en/infraops/infrastructure-metrics.asciidoc]: Not the biggest deal, but you start out using "maximum", then switch to "max" from here on. Any reason why?

File: docs/en/infraops/installation.asciidoc [hunk 4]
```diff
@@ -1,165 +1,120 @@
...
-* Follow the instructions in the Add Data section of the {kib} home page. Click
-*Add log data* or *Add metrics*, and follow the links for the types of data you
-want to collect.
-+
-[role="screenshot"]
-image::add-data.png[]
+[float]
+==== Install {beats} from {kib}
+
+To install {beats} from {kib}, on the machine where you want collect the data, open a {kib} browser window.
```
Comment (bmorelli25) [inline docs/en/infraops/installation.asciidoc]:
```suggestion
To install {beats} from {kib}, on the machine where you want to collect the data, open a {kib} browser window.
```

File: docs/en/infraops/infrastructure-metrics.asciidoc (hunk 2 above)
Comment (Titch990) [reply docs/en/infraops/infrastructure-metrics.asciidoc]: oops!

File: docs/en/infraops/infrastructure-metrics.asciidoc (hunk 3 above)
Comment (Titch990) [reply docs/en/infraops/infrastructure-metrics.asciidoc]: Good spot!

File: docs/en/infraops/infrastructure-metrics.asciidoc [hunk 5]
```diff
@@ -0,0 +1,54 @@
+[[infrastructure-metrics]]
+[role="xpack"]
+
+== Infrastructure metrics
+
+The metrics listed below are provided by the {beats} shippers.
+// ++ what does the sentence below mean? What does the reader need to do?
```
[the remark this answers is not in the document]
Comment (Titch990) [reply docs/en/infraops/infrastructure-metrics.asciidoc]: Thanks. This topic needs more work. For now, I'll update my comment, and address the issue when I revisit the topic.

File: docs/en/infraops/infrastructure-metrics.asciidoc [hunk 6]
```diff
@@ -0,0 +1,54 @@
...
+Each system type requires their corresponding identity field to be in the same event document:
+
+* Hosts require `host.name`
+* Docker containers require `container.id`
+* Kubernetes pods require `kubernetes.pod.uid`
+
+// + I think this bit applies to the View Metrics page, not the Infrastructure home page.
+// ++ What does this mean, and what does the reader need to do about it, if anything to ensure the "required field" is available?
```
[the remark this answers is not in the document]
Comment (Titch990) [reply docs/en/infraops/infrastructure-metrics.asciidoc]: as above.

File: docs/en/infraops/installation.asciidoc [hunk 7]
```diff
@@ -1,165 +1,120 @@
...
-===== Kubernetes Pod Metrics
-
-*CPU Usage*:: Average of `kubernetes.pod.cpu.usage.node.pct`
-
-*Memory Usage*:: Average of `kubernetes.pod.memory.usage.node.pct`
-
-*Inbound Traffic*:: Derivative of the max of `kubernetes.pod.network.rx.bytes` scaled to a 1 second rate
-
-*Outbound Traffic*:: Derivative of the max of `kubernetes.pod.network.tx.bytes` scaled to a 1 second rate
+// ++ What does this mean?
```
[the remark this answers is not in the document]
Comment (Titch990) [reply docs/en/infraops/installation.asciidoc]: Thanks. That's clearer now.

**Pull request merged**